app/views/candidate_mailer/confirm_sign_in.text.erb

```diff
@@ -0,0 +1,11 @@
+# New sign in to Apply for teacher training
+
+We detected you have signed in to Apply for teacher training from a new device.
+
+# New device details
+IP address: <%= @device[:ip_address] %>
+Browser: <%= @device[:user_agent] %>
+
+If this was you, you can ignore this email.
+
+If this was not you, contact us immediately on the number below.
```

Comment:
for consistency?

```suggestion
If this was not you, please contact us immediately.
```

Reply:
Hey Duncan,

@Tomcowan1 has made some revisions to the content on the candidate email. Should the provider and candiate emails be the same?

Happy to make the suggested changes to the provider content for the time being

Reply:
Ha — I'm not a content designer. But at least if we're saying "please" to providers we can probably say it to candidates too?